website/content/_index.md

```diff
@@ -5,7 +5,7 @@ description = ""
 draft= false
 +++
 
-Wash helps you deal with all your remote or cloud-native infrastructure using the UNIX-y patterns and tools you already know and love.
+Wash helps you manage a bunch of different things using well-established UNIX-y patterns and tools to free you from having to learn multiple ways of doing the same thing.
```

Comment:
I think this phrasing's OK. Open to better ways of stating this. My main goal here is to make it familiar for newer people (which is why I removed the "you already know and love" bit) while also highlighting the main problem Wash is trying to solve (to free you from having to learn multiple ways of doing the same thing).

Reply:
Re #1, I think we need to be more specific than "a bunch of different things". The thing I like about remote / cloud native is that it describes the actual plugins we've got currently.

I'd also change "from having to learn" (learning is a good thing, i figure!) to something else...maybe "from having to remember"? "having to deal with"?

I think we should also add a clause (separate sentence) around letting folks easily build higher-level tools, instead of wrestling with multiple, lower-level ones.